helm-chart/templates/nextcloud-aio-apache-deployment.yaml

```diff
@@ -16,11 +16,14 @@ spec:
     metadata:
       annotations:
         kompose.cmd: kompose convert -c -f latest.yml
-        kompose.version: 1.27.0 (b0ed6a2c9)
+        kompose.version: 1.28.0 (c4137012e)
       labels:
         io.kompose.network/nextcloud-aio: "true"
         io.kompose.service: nextcloud-aio-apache
     spec:
+      securityContext:
+        fsGroup: 65534
+        fsGroupChangePolicy: "OnRootMismatch"
```

Comment:
@mrueg do you know if this is the right way to solve possible permission issues with the volumes? I suppose kubernetes is not able to just use the correct permissions automatically that get applied inside a container image like docker volumes do?

Reply:
Maybe @st3iny or @provokateurin know?

Reply:
Seems like this should work. Testing will show.